src/task_4_2_get_ranges.py

```diff
@@ -0,0 +1,55 @@
+"""Task_4_2."""
+
+from itertools import groupby
+
+
+class OneFromLast():
+    """."""
+
+    def __init__(self, first):
+        """."""
+        self.prev = first
+        self.gr_id = first
+
+    def __call__(self, value):
+        """."""
+        changed = +(value - self.prev) != 1
+        self.prev = value
+        if changed:
+            self.gr_id = value
+        return self.gr_id
+
+
+def get_ranges(arr):
+    """Decription.
+
+    Реализовать функцию get_ranges которая получает на вход непустой
+    список неповторяющихся целых чисел, отсортированных по возрастанию,
+    которая этот список “сворачивает”
+    get_ranges([0, 1, 2, 3, 4, 7, 8, 10]) // "0-4,7-8,10"
+    get_ranges([4,7,10]) // "4,7,10"
+    get_ranges([2, 3, 8, 9]) // "2-3,8-9"
+    """
+    ranges = ''
+    for gr_id, values in groupby(arr, OneFromLast(arr[0])):
```

Comment:
it looks overhead here. It doesn't became easier to read or lesser because of using lib so why do we use it?